ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -383,13 +396,20 @@ try:
                                                        'aki_i386',
                                                        '--boot-kernel')
                 except:
-                    print >> sys.stderr, 'Could not find bootkernel in config'
+                    print(
+                        'Could not find bootkernel in config',
+                        file=sys.stderr
+                    )
                     sys.exit(1)
             else:
-                print >> sys.stderr, 'Could not reliable determine the ',
-                print >> sys.stderr, 'bootkernel to use '
-                print >> sys.stderr, 'must specify bootkernel, ',
-                print >> sys.stderr, 'arch (x86_64|i386) or hvm'
+                print(
+                    'Could not reliable determine the ',
```

Comment:
TODO: replace "reliable" with "reliably" (not really apropos to this PR - just noticed it.)